joomla/joomla-cms
# Tinymce code icon in simple view

### Summary of Changes
### Testing Instructions

Switch tinymce into simple functionality
you will see the code editor AFTER the xtd-plugins

Switch tinymce into advanced functionality
you will see the code editor BEFORE the xtd-plugins with the link unlink icons

After the PR the icon will be before the xtd-plugins - see screenshots
#### Before

![cmqv](https://cloud.githubusercontent.com/assets/1296369/17771983/1bf11462-653e-11e6-88db-d35e745da6fd.png)
#### After

![09z 2](https://cloud.githubusercontent.com/assets/1296369/17771951/f4ade376-653d-11e6-88ca-1272bbd4fd81.png)
